drivers/wifi/nrf700x/osal/fw_if/umac_if/src/cmd.c

```diff
@@ -160,6 +160,14 @@ enum wifi_nrf_status umac_cmd_init(struct wifi_nrf_fmac_dev_ctx *fmac_dev_ctx,
 	umac_cmd_data->temp_vbat_config_params.vth_very_low = NRF_WIFI_VBAT_VERYLOW;
 #endif /* !CONFIG_NRF700X_RADIO_TEST */
 
+	wifi_nrf_osal_mem_cpy(fmac_dev_ctx->fpriv->opriv,
+			      umac_cmd_data->set_reg_info.nrf_wifi_alpha,
+			      CONFIG_NRF700X_REG_DOMAIN,
+			      (sizeof(umac_cmd_data->set_reg_info.nrf_wifi_alpha)));
```

Comment:
NRF_WIFI_COUNTRY_CODE_LEN can be used instead of (sizeof(umac_cmd_data->set_reg_info.nrf_wifi_alpha)) to reduce the length of code and to make it more readable.